doc/base/classes.xml

```diff
@@ -44482,21 +44495,26 @@
 			<argument index="0" name="base_object" type="Object">
 			</argument>
 			<description>
-				Return true if a given object uses an instance of this script.
+				Returns true if 'base_object' is an instance of this script.
 			</description>
 		</method>
 		<method name="is_tool" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns true if the script is a tool script. A tool script can run in the editor.
+				To enable tool mode in GDScript, place the [code]tool[/code] keyword at the top of the file.
 			</description>
 		</method>
 		<method name="reload">
 			<return type="int" enum="Error">
 			</return>
 			<argument index="0" name="keep_state" type="bool" default="false">
+				If true, preserve existing script instances and subclasses.
 			</argument>
 			<description>
+				Compiles the script from source code. Returns an error code, but raises an error if the source code contains a syntax error.
+				Returns ERR_ALREADY_IN_USE if instances of the script exist and 'keep_state' is false.
```

Comment:
The behavior described here is not mandated. Not every script implementation triggers compilation or parsing from this method.

Reply:
Okay, I'll simplify it to "Reloads the script's class implementation. Returns an error code." There's not much else to say since this function isn't implemented for other types of scripts yet.

Is it obvious that you shouldn't reload a script while it is being executed, or is it worth mentioning that here?